api/mysagw/case/serializers.py

```diff
@@ -72,3 +73,36 @@ def create(self, validated_data):
         )
 
         return instance
+
+
+class CaseTransferSerializer(drf_serializers.Serializer):
+    removable_assignees = drf_serializers.ListField(
+        child=drf_serializers.PrimaryKeyRelatedField(queryset=Identity.objects.all())
+    )
+    new_assignees = drf_serializers.ListField(
+        child=drf_serializers.PrimaryKeyRelatedField(queryset=Identity.objects.all()),
+        allow_empty=False,
+    )
+    case_ids = drf_serializers.ListField(
+        child=drf_serializers.UUIDField(), allow_empty=False
+    )
+
+    def create(self, validated_data):
+        new_accesses = []
+        for case_id in validated_data["case_ids"]:
+            for new_assignee in validated_data["new_assignees"]:
+                if not models.CaseAccess.objects.filter(
+                    identity=new_assignee, case_id=case_id
+                ).exists():
+                    new_accesses.append(  # noqa: PERF401
+                        models.CaseAccess.objects.create(
+                            case_id=case_id, identity=new_assignee
+                        )
+                    )
+
+        to_remove = models.CaseAccess.objects.filter(
+            case_id__in=validated_data["case_ids"],
+            identity__in=validated_data["removable_assignees"],
```

Comment:
If it receives no `removable_assignees` it would just be a bulk a addition of accesses, right?
Maybe would make sense to add a condition to check if there are any to remove.

Reply:
Done :+1: